app/population/controllers/edit/details.js

```diff
@@ -80,5 +73,19 @@ class DetailsController extends FormWizardController {
 
     next()
   }
+
+  setHintText(req, res, next) {
+    if (!req.population) {
```

Comment:
If there is no population, then this is "Add numbers" - so we'll need to dynamically change the hint text.

The alternative would be to set the hint text by default, and delete it when in "Update numbers" mode.